src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -13,6 +13,10 @@ case class ThurloeKeyValue(
   userId: Option[String] = None,
   keyValuePair: Option[FireCloudKeyValue] = None)
 
+case class ThurloeKeyValues(
```

Comment:
Cool, I'm down for that as a follow-up. I'll merge this as soon as the next alpha snapshot is taken